src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java

```diff
@@ -2938,4 +2938,72 @@ public float getItemPaneOffsetFactor() {
 	public float getInfoPaneOffsetFactor() {
 		return infoPaneOffsetFactor.getValue() * getWidthMult();
 	}
+
+	public ResourceLocation getCustomEquipmentTexture(boolean isPetRendering) {
+		switch (NotEnoughUpdates.INSTANCE.config.customArmour.colourStyle) {
+			case 0:
+				return ARMOR_DISPLAY;
+			case 1:
+				return ARMOR_DISPLAY_GREY;
+			case 2:
+				return ARMOR_DISPLAY_DARK;
+			case 3:
+				return isPetRendering ? ARMOR_DISPLAY_TRANSPARENT_PET : ARMOR_DISPLAY_TRANSPARENT;
+			case 4:
+				return ARMOR_DISPLAY_FSR;
+		}
+		return null;
+	}
+
+	public ResourceLocation getCustomPetTexture(boolean isArmorRendering) {
+		switch (NotEnoughUpdates.INSTANCE.config.petOverlay.colourStyle) {
+			case 0:
+				return isArmorRendering ? PET_ARMOR_DISPLAY : PET_DISPLAY;
+			case 1:
+				return isArmorRendering ? PET_ARMOR_DISPLAY_GREY : PET_DISPLAY_GREY;
+			case 2:
+				return isArmorRendering ? PET_ARMOR_DISPLAY_DARK : PET_DISPLAY_DARK;
+			case 3:
+				return isArmorRendering ? PET_ARMOR_DISPLAY_TRANSPARENT : PET_DISPLAY_TRANSPARENT;
+			case 4:
+				return isArmorRendering ? PET_ARMOR_DISPLAY_FSR : PET_DISPLAY_FSR;
+		}
+		return null;
+	}
```

Comment:
Am i seeing this wrong or are you duplicating the code from https://github.com/NotEnoughUpdates/NotEnoughUpdates/pull/241 in a different class (while also reintroducing the half pixel offset bug a few lines below)

Reply:
yes

Reply:
Please revert that and merge #241 into this then

Reply:
it's a better idea. than what I had in mind

Reply:
https://github.com/Ascynx/NotEnoughUpdates/commit/edc6053edf12ac1fbd8ae232385d3cd69702ebad should be what this one isn't. I know I'm bad at using github no need to remind me

Reply:
moved to https://github.com/NotEnoughUpdates/NotEnoughUpdates/pull/251 as this one is too much of a mess